### ISSUE TITLE
new bug in 1.2.1 - disable mouse accel only works after toggling checkbox on and off again

### ISSUE THREAD

**Issue by luckman212** [ther0n/UnnaturalScrollWheels#20]
- macOS 11.2

I noticed the [`disableMouseAccel()`](https://github.com/ther0n/UnnaturalScrollWheels/blob/master/UnnaturalScrollWheels/AppDelegate.swift#L118) func got some work in 1.2.1

Upon launch, regardless of the preferences (my prefs below) - mouse acceleration is **disabled**. It looks like the following code needs to be run in `refresh()` but instead it's only running in the `disableMouseAccelClicked()` IBAction.

```swift
if Options.shared.disableMouseAccel {
    Options.shared.accel = -1
} else {
    Options.shared.accel = Options.shared.origAccel
}
```

my prefs:
![image](https://user-images.githubusercontent.com/1992842/106901265-87ed5300-66c5-11eb-85dd-338715dc94e0.png)

```
$ defaults read com.theron.UnnaturalScrollWheels
{
    AlternateDetectionMethod = 0;
    DisableMouseAccel = 0;
    DisableScrollAccel = 0;
    FirstLaunch = 0;
    InvertHorizonalScroll = 0;
    InvertHorizontalScroll = 0;
    InvertVerticalScroll = 1;
    "NSStatusItem Preferred Position Item-0" = 6128;
    "NSStatusItem Preferred Position Item-1" = 6058;
    OriginalAccel = 45056;
    ScrollLines = 3;
    ShowMenuBarIcon = 1;
}
```

**Comment by ther0n** [comment]
Shoot you're right. I missed that when testing since I like having mouse acceleration off myself. Thanks for doing some digging, I'll get a fix out in a couple hours during my lunch.

**Comment by ther0n** [comment]
Fixed in [1.2.2](https://github.com/ther0n/UnnaturalScrollWheels/releases/tag/1.2.2)

**Comment by luckman212** [comment]
Amazing, thank you!
